projects/core/styles/components/link.less

```diff
@@ -23,7 +23,7 @@
     cursor: pointer;
     font: inherit;
     color: inherit;
```

Comment:
Let's set primary color here, instead of `[tuiTheme='dark'] &[data-appearance=''] {`